extra/utils/Validation.js

```diff
@@ -1,48 +1,24 @@
-let users = [
-    {
-        traineeEmail : 'shashank.baranawal@successive.tech',
-        reviewerEmail : 'naman.parashar@successive.tech',
-    },
-    {
-        traineeEmail : 'shashank.baranwal@gmail.com',
-        reviewerEmail : 'naman.parashar@gmail.com'
-    }
-]
-
-function validateEmail(email){
-    const regix = /\w+.\w+@successive.tech$/i;
-    return regix.test(email)
-}
-
-
+import validateEmail from './helpers'
 //console.log(validateEmail(users[0]["traineeEmail"]))
-let a =[];
-let b =[];
-function validateUsers(users){
+// let a =[];
+// let b =[];
+export default function validateUsers(users){
+
     let countValid=0;
     let countInvalid=0;
-    users.forEach(item => {
-        const {traineeEmail,reviewerEmail}= item;
-        if(validateEmail(traineeEmail)){
-            a.push(traineeEmail);
+    users.forEach(function(item){
+        if(validateEmail(item.traineeEmail)===true && validateEmail(item.reviewerEmail)===true){
             countValid++;
         }
         else{
-            b.push(traineeEmail);
-            countInvalid++;
-        }
-        if(validateEmail(reviewerEmail)){
-            a.push(reviewerEmail);
-            countValid++;
-        }
-        else{
-            b.push(reviewerEmail);
             countInvalid++;
         }
     })
```

Comment:
Create the function as per instruction in task, do not change the approved code

Reply:
this issues will not happen again.